Arrowgene.Ddon.GameServer/Chat/Command/Commands/TimeCommand.cs

```diff
@@ -0,0 +1,53 @@
+using Arrowgene.Ddon.Database.Model;
+using Arrowgene.Ddon.GameServer.Handler;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Arrowgene.Ddon.GameServer.Chat.Command.Commands
+{
+    public class TimeCommand : ChatCommand
+    {
+        public override AccountStateType AccountState => AccountStateType.User;
+        public override string Key => "time";
+        public override string HelpText => "usage: `/time` - Print details about the time/weather/moon";
+
+        private DdonGameServer _server;
+
+        public TimeCommand(DdonGameServer server)
+        {
+            _server = server;
+        }
+
+        public override void Execute(string[] command, GameClient client, ChatMessage message, List<ChatResponse> responses)
+        {
+            StringBuilder sb = new StringBuilder();
+
+            long OriginalRealTimeSec = 0x55DDD470;
```

Comment:
Should you make a constant for this value `0x55DDD470` since it is used in a few places?